examples/megacli.cpp

```diff
@@ -4730,6 +4876,150 @@ static void process_line(char* l)
 
                         }
                     }
+                    else if (words[0] == "chatlu")
+                    {
+                        if (words.size() == 2)
+                        {
+                            handle publichandle;
```

Comment:
If `handle` is not initialized to zero, `Base64::atob`, depending on the last parameter, it could not set all bytes. There are more cases of this possible problem above and below.

Reply:
Nice catch. Thanks for the notice. It's also interesting for @jgandres. I've already fixed it.